apis/frontend/implementation-guide-session-api/advanced-usage/spa-out-out-of-session-tracking.md

```diff
@@ -55,4 +55,4 @@ This section lists the impact of enabled `doNoTrack` mode on various Nosto featu
 | Recommendations and Onsite Content Personalization  | Normally served                                  |
 | Popups                                          | Fully disabled                                  |
 | Search and Category Merchandising (Universal)   | Analytics tracking disabled                     |
-| Category Merchandising (Platform)               | Segmentation capabilities are fully disabled    |
+| Category Merchandising (Platform)               | Segmentation logic is disabled                  |
```

Comment:
Maybe rephrase it so that it's easy to understand for non-developers? Specifically state what's the impact on the analytics and segmentation?

Reply:
Updated